test/e2e/windows/gmsa_full.go

```diff
@@ -411,8 +481,17 @@ func createPodWithGmsa(f *framework.Framework, serviceAccountName string) string
 			},
 		},
 	}
-	f.PodClient().CreateSync(pod)
 
+	if testFileWritting {
```

Comment:
what about move this parameter/command field to outside the pod spec creation, makes more sense to bring this into the test as another `runKubectlExecInNamespace` command

Reply:
Sure!